### DISCUSSION TITLE
How to change hlsearch color?

### DISCUSSION THREAD

**Discussion by champignoom** [neovim/neovim#31545] [Q&A]
![image](https://github.com/user-attachments/assets/89e6a467-e81d-4b81-a525-6bda448e6894)

Under the default colorscheme and dozens of others, with `hlsearch` enabled, the current hit is much less highlighted than the rest of the matches, which makes it rather impossible to locate visually. Try to locate the currently matched `a` in the image above.

Does anyone know how to change the highlight of current hit?

**Comment by zeertzjq** [comment]
Try changing the value of the `'guicursor'` option.
